PsychSourceGL/Cohorts/FTGLTextRenderer/libptbdrawtext_ftgl.cpp

```diff
@@ -450,11 +449,7 @@ void PsychSetTextBGColor(int context, double* color)
 // renderbackend needs a different geometric setup:
 void PsychSetTextViewPort(int context, double xs, double ys, double w, double h)
```

Comment:
If it works, yes. We just have to be consistent across all potential future plugins, and so far the ViewPort setting wasn't doing anything spectacular, so should be fine. I'll only test this pull sometime later in the next release cycle.

Btw. i think some of the improvements you made wrt. building the plugin on Windows don't actually work with my msvc 2010 build environment. Unfortunately i can't remember what the problem was, it was already a couple of months ago, but i'll probably have to revert that commit until we upgrade to a more modern msvc version - and the resulting change in msvc runtime library versions etc.